webpack/dlls/README.md

```diff
@@ -0,0 +1,31 @@
+# DLLs
+
+Webpack "DLLs" can help reduce the build times a bit which is rather useful
+during development.
+
+You may opt-in to use the Vendor DLL by doing the following:
+
+- set `WEBPACK_DLLS=1` in your shell before running webpack
+- compile the DLL using `npm run build-dlls`
```

Comment:
You don't need the flag when you're building using `npm run build-dlls` but you do need it set to _use_ the DLLs, which is `npm run dev` and friends.

I personally think DLLs should only be used during development and had outlined my reasons in the main thread but it's not necessarily a must. I just don't see a gain in using it in a production environment as the prod build is already expected to be slow and doesn't run nearly as often as the dev one so.